app/Http/Controllers/PermissionGroupController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+
+use App\Repositories\Admin\PermissionGroup\PermissionGroupRepository;
+use App\Repository\PermissionGroupRepositoryInterface;
+use  App\Http\Requests\Admin\PermissionGroupRequest;
+
+
+class PermissionGroupController extends Controller
+{
+    protected $permissionGroupRepository;
+
+    public function __construct(PermissionGroupRepositoryInterface $permissionGroupRepository)
+    {
+        $this->permissionGroupRepository = $permissionGroupRepository;
+    }
+
+    public function index(){
+        return view('admin.permission_group.index', [
+            'permissionGroups' => $this->permissionGroupRepository->paginate()
+        ]);
+    }
+
+    public function create()
+    {
+        return view('admin.permission_group.form');
+    }
+
+    public function store(PermissionGroupRequest $request)
+    {
+        $this->permissionGroupRepository->save($request->all());
+        return redirect()->route('permission_group.index');
+    }
+
+    public function show($id)
+    {
+        if (!$permissionGroup = $this->permissionGroupRepository->findById($id))
```

Comment:
```
if (!$permissionGroup = $this->permissionGroupRepository->findById($id))
        {
        abort(404);
        }
```

=>

if (!$permissionGroup = $this->permissionGroupRepository->findById($id)) {
        abort(404);
}